control.go

```diff
@@ -37,15 +37,15 @@ type Control struct {
 }
 
 type ControlHostInfo struct {
```

Comment:
Its probably time to ditch this and just export the `cert.Certificate` interface

Reply:
A change for another time